### DISCUSSION TITLE
XQUIC1.8.0无法与tengine兼容问题

### DISCUSSION THREAD

**Discussion by ruanqingxuan** [alibaba/xquic#450] [Q&A] [answered]
xquic1.8.0添加了fec等功能后，与tengine无法兼容，导致tengine开启了之后用浏览器访问一直导致网页加载失败是怎么回事

**Comment by Yanmei-Liu** [comment, marked as answer]
目前tengine社区还未在module中更新FEC相关功能，因此在tengine module使用时，可先关闭FEC编译宏的方式关闭所有FEC相关功能，以保证兼容性。FEC编译宏包括以下参数，默认为不包含（即不带就是不包含），在tengine中使用时可以删除以下参数：

> -DXQC_ENABLE_FEC=1 -DXQC_ENABLE_XOR=1 -DXQC_ENABLE_RSC=1 -DXQC_ENABLE_PKM=1
